server/tests/rest/shelter/test_shelter.py

```diff
@@ -0,0 +1,108 @@
+import json
+import pytest
+import sys
+import os
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../..")))
```

Comment:
```suggestion
```

Reply:
we don't need that, because we use $PYTHONPATH environment variable to configure pytest